desktop/apps/artwork/lib/backbone_react_view/README.md

```diff
@@ -0,0 +1 @@
+#### This is an experiment in modeling declarative backbone views. WIP for a later point in time
```

Comment:
Ignore the following for now -- an experiment in thinking about possible future migration paths.